src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const rows = Array.from(table.querySelectorAll('tbody tr'));
+
+headers.forEach((header, columnIndex) => {
```

Comment:
instead of adding an event listener to each header separately it's better to use [event delegation](https://www.freecodecamp.org/news/event-delegation-javascript/). Event bubbling allows to do it

FYI